### ISSUE TITLE
Component: Calendar invalid aria-controls

### ISSUE THREAD

**Issue by mristov** [primefaces/primeng#15004]
### Describe the bug

![image](https://github.com/primefaces/primeng/assets/150306240/aa33e3d5-472e-4177-9086-96b372697784)

It is only happening when we open p-calendar, otherwise this issue is not showing

### Environment

PrimeNg 17.10

### Angular version

17.0.5

### PrimeNG version

17.10

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.16.1

**Comment by TaneliTuomola** [comment]
Duplicate of https://github.com/primefaces/primeng/issues/14980
Id attribute is missing from the date picker overlay element.
